mule-extensions-api/src/main/java/org/mule/runtime/extension/api/model/AbstractComponentModel.java

```diff
@@ -35,15 +41,16 @@
   /**
    * Creates a new instance
    *
-   * @param name                 the model's name
-   * @param description          the model's description
+   * @param name the model's name
+   * @param description the model's description
```

Comment:
Fix the comments indentation 😄 (as was before)

Reply:
will do